coderd/externalauth/externalauth.go

```diff
@@ -886,6 +895,8 @@ func bitbucketServerDefaults(config *codersdk.ExternalAuthConfig) codersdk.Exter
 		DisplayName: "Bitbucket Server",
 		Scopes:      []string{"PUBLIC_REPOS", "REPO_READ", "REPO_WRITE"},
 		DisplayIcon: "/icon/bitbucket.svg",
+		// TODO: PKCE support? Test 'S256' as the string value
```

Comment:
I get the gist, but these TODOs could be a bit clearer, if someone else needs to look at it in the future.